org.civicrm.bootstrapcivihr/scss/components/outside-namespace/_h-scroll-shadows.scss

```diff
@@ -0,0 +1,42 @@
+.chr_h-scroll-shadows__master {
+  overflow: hidden; // this ensures the shadows are hidden behind
+  position: relative;
+}
+
+.chr_h-scroll-shadows__wrapper {
+  overflow-x: auto;
+
+  &::before,
+  &::after {
+    box-shadow: 0 0 10px 1px rgba(48, 40, 40, 0.5);
+    content: '';
+    display: block;
+    height: calc(100% - 10px); // if height is 100% then the shadow will be cut off
+    opacity: 0; // by default the shadows are invisible
```

Comment:
i think with better names for the element (see above), this comment is superfluous